packages/wrangler/src/__tests__/mtls-certificates.test.ts

```diff
@@ -354,52 +354,6 @@ describe("wrangler", () => {
 		});
 
 		describe("commands", () => {
-			describe("help", () => {
```

Comment:
Instead of removing this test, could you fix the typo so it's testing what the test name describes?